src/docs/framework-integration/angular.md

```diff
@@ -125,15 +164,19 @@ import { DIRECTIVES } from './stencil-generated';
 export class ExampleLibraryModule {}
 ```
 
-Any components that are included in the `exports` array should additionally be exported in your main entry point (either `public-api.ts` or `index.ts`). Skipping this step will lead to Angular Ivy errors when building for production.
+Any components that are included in the `exports` array should additionally be exported in your main entry point (either `public-api.ts` or
+`index.ts`). Skipping this step will lead to Angular Ivy errors when building for production.
 
 ```ts
 export { DIRECTIVES } from './stencil-generated';
 ```
 
-### Link your packages (optional)
+> **NOTE:** The default behavior does not automatically define the Web Components needed by the proxy components. You can manually define the Web
+> Components following the [documentation for the dist output target](/docs/custom-elements#defining-exported-custom-elements).
+
+### Link Your Packages (Optional)
 
-> If you are using a monorepo tool (Lerna, Nx), skip this section.
+> If you are using a monorepo tool (Lerna, Nx, etc.), skip this section.
```

Comment:
this wasn't my experience, at least entering the commands as described in the tutorial right now.

I think that we'd have to modify the init commands to put both the Stencil component library and the Angular thingy in `packages/` in order for lerna to do this automatically. As is, I had to do `npm link` in the Stencil directory and then `npm link stencil-library` in the angular directory

Reply:
Yeah, I ran into this too actually with the React stuff. I'll make the same changes I had to here